From 08e0f26de458cee21850ae1fd635a15fde950db3 Mon Sep 17 00:00:00 2001
From: Tosone <i@tosone.cn>
Date: Sun, 5 Jan 2025 12:30:39 +0800
Subject: [PATCH 1/2] fix typo

---
 src/module.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/module.c b/src/module.c
index 3256234cc..dee98da90 100644
--- a/src/module.c
+++ b/src/module.c
@@ -33,9 +33,9 @@
 #include "arithmetic/arithmetic_expression.h"
 
 // minimal supported Redis version
-#define MIN_REDIS_VERION_MAJOR 7
-#define MIN_REDIS_VERION_MINOR 2
-#define MIN_REDIS_VERION_PATCH 0
+#define MIN_REDIS_VERSION_MAJOR 7
+#define MIN_REDIS_VERSION_MINOR 2
+#define MIN_REDIS_VERSION_PATCH 0
 
 static int _RegisterDataTypes(RedisModuleCtx *ctx) {
 	if(GraphContextType_Register(ctx) == REDISMODULE_ERR) {
@@ -120,12 +120,12 @@ int RedisModule_OnLoad
 	});
 
 	// validate minimum redis-server version
-	if(!Redis_Version_GreaterOrEqual(MIN_REDIS_VERION_MAJOR,
-				MIN_REDIS_VERION_MINOR, MIN_REDIS_VERION_PATCH)) {
+	if(!Redis_Version_GreaterOrEqual(MIN_REDIS_VERSION_MAJOR,
+				MIN_REDIS_VERSION_MINOR, MIN_REDIS_VERSION_PATCH)) {
 		RedisModule_Log(ctx, "warning",
 				"FalkorDB requires redis-server version %d.%d.%d and up",
-				MIN_REDIS_VERION_MAJOR, MIN_REDIS_VERION_MINOR,
-				MIN_REDIS_VERION_PATCH);
+				MIN_REDIS_VERSION_MAJOR, MIN_REDIS_VERSION_MINOR,
+				MIN_REDIS_VERSION_PATCH);
 		return REDISMODULE_ERR;
 	}
 
-- 
2.47.1

